src/plugins/auto/executable_network.cpp

```diff
@@ -797,16 +797,25 @@ InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetMetric(const std::st
                     real = defaultNumForLatency;
                 }
             }
-            IE_SET_METRIC_RETURN(OPTIMAL_NUMBER_OF_INFER_REQUESTS, real);
+            return decltype(ov::optimal_number_of_infer_requests)::value_type {real};
         }
 
         if (_loadContext[ACTUALDEVICE].isAlready) {
             return _loadContext[ACTUALDEVICE].executableNetwork->GetMetric(name);
         }
         return _loadContext[CPU].executableNetwork->GetMetric(name);
     }
+    if (name == ov::supported_properties) {
+        return decltype(ov::supported_properties)::value_type {
+            // Metrics
+            ov::PropertyName{ov::supported_properties.name(), ov::PropertyMutability::RO},
```

Comment:
how about model_priority?

Reply:
model_priority is metric for plugin:
https://github.com/openvinotoolkit/openvino/blob/fa81a300937e873c3c331b2e9992f5ae736a2fe8/src/plugins/auto/plugin.cpp#L64

Reply:
it can be property for both plugin and compiled model:
1. For plugin: RW
2. For compiled model: RO

Reply:
model_priority in AUTO is used for loading different network to different device. 

This key will use in AUTO only and not pass to target device plugin like GPU plugin.

Reply:
Hi ilya @ilya-lavrenov , do you mean for Compiled Model, when get Metric get_property with ov::supported_properties, the value are all RO? I was referring to gpu execnetwork below:
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/plugin/compiled_model.cpp#L157